app/models/customer.py

```diff
@@ -0,0 +1,30 @@
+from app import db
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    videos_checked_out_count = db.Column(db.Integer, default=0, nullable=True)
+
```

Comment:
You can also add relationships here  so you don't need to have `db.query` in your routes file.

```suggestion
    rentals = db.relationship('Rental',backref = 'customer')
```

With this you can have `my_customer.rentals` to get a list of rentals for a specific customer.